--- netatalk-1.7cvs/bin/afile/Makefile.in.orig	2003-01-07 20:44:23.000000000 -0500
+++ netatalk-1.7cvs/bin/afile/Makefile.in	2003-01-08 07:09:05.000000000 -0500
@@ -117,12 +117,12 @@
 install_sh = @install_sh@
 
 bin_PROGRAMS = afile achfile
-bin_SCRIPTS = acleandir.rc
+bin_SCRIPTS = # acleandir.rc
 
 afile_SOURCES = afile.c common.c common.h
 achfile_SOURCES = achfile.c common.c common.h
 
-EXTRA_DIST = acleandir.rc
+EXTRA_DIST = # acleandir.rc
 subdir = bin/afile
 mkinstalldirs = $(SHELL) $(top_srcdir)/mkinstalldirs
 CONFIG_HEADER = $(top_builddir)/config.h
