--- netatalk.cvs/contrib/printing/add_netatalk_printer
+++ netatalk.debian/contrib/printing/add_netatalk_printer
@@ -43,10 +43,10 @@
 
 # allow for the env NETATALKHOME to override the guessed one from above
 
-NETATALKHOME=${NETATALKHOME:-$RUNHOME}
+NETATALKHOME=${NETATALKHOME:-/usr}
 export NETATALKHOME
 
-PATH=/bin:${PATH}:${NETATALKHOME}/bin:${NETATALKHOME}/etc:${NETATALKHOME}/etc/filters:/usr/lib:/usr/sbin
+PATH=/bin:${PATH}:${NETATALKHOME}/bin:${NETATALKHOME}/etc:${NETATALKHOME}/lib/atalk/filters:/usr/lib:/usr/sbin
 
 if [ ! -x ${NETATALKHOME}/bin/pap ]; then
        echo "OOPS:     I don't see ${NETATALKHOME}/bin/pap ,"
@@ -66,7 +66,7 @@
 
 echo ""
 echo "Looking for LaserWriters in Zone ${ZONE} ..."
-$NETATALKHOME/bin/nbplkup ":LaserWriter@${ZONE}"
+${NETATALKHOME}/bin/nbplkup ":LaserWriter@${ZONE}"
 
 echo ""
 echo "Enter AppleTalk printer name: \c"
@@ -80,7 +80,7 @@
 
 echo "checking nbplkup ${DEST}:LaserWriter@${ZONE}"
 echo ""
-TestDEST=`$NETATALKHOME/bin/nbplkup "${DEST}:LaserWriter@${ZONE}"`
+TestDEST=`${NETATALKHOME}/bin/nbplkup "${DEST}:LaserWriter@${ZONE}"`
 echo "${TestDEST}"
 echo ""
 
@@ -237,7 +237,7 @@
 Printer types: Netatalk
 Printers: any
 Filter type: fast
-Command: ${NETATALKHOME}/etc/filters/ifpap 2>&1 > /dev/null
+Command: ${NETATALKHOME}/lib/atalk/filters/ifpap 2>&1 > /dev/null
 EOF
        chown lp:lp /etc/lp/fd/netatalk.fd
        chmod 664   /etc/lp/fd/netatalk.fd
@@ -257,7 +257,7 @@
 Printer types: Netatalk-R
 Printers: any
 Filter type: fast
-Command: "/usr/lib/lp/postscript/postreverse | ${NETATALKHOME}/etc/filters/ifpap 2>&1 >/dev/null"
+Command: "/usr/lib/lp/postscript/postreverse | ${NETATALKHOME}/lib/atalk/filters/ifpap 2>&1 >/dev/null"
 EOF
        chown lp:lp /etc/lp/fd/netatalk-r.fd
        chmod 664   /etc/lp/fd/netatalk-r.fd
