--- netatalk.cvs/config/netatalk.pamd
+++ netatalk.debian/config/netatalk.pamd
@@ -1,6 +1,6 @@
 #%PAM-1.0
-auth       required    /lib/security/pam_unix.so
-account    required    /lib/security/pam_unix.so 
-#password   required   /lib/security/pam_cracklib.so
-#password   required   /lib/security/pam_unix.so use_authtok
-session    required    /lib/security/pam_unix.so 
+auth       required    pam_unix.so
+account    required    pam_unix.so 
+password   required    pam_cracklib.so
+password   required    pam_unix.so use_authtok
+session    required    pam_unix.so 
