--- netatalk.cvs/include/atalk/paths.h
+++ netatalk.debian/include/atalk/paths.h
@@ -24,7 +24,7 @@
 #endif
 #else
 #ifdef linux
-#define _PATH_LOCKDIR           "/var/lock/"
+#define _PATH_LOCKDIR           "/var/run/"
 #else
 #define _PATH_LOCKDIR           "/var/spool/locks/"
 #endif /* linux */
@@ -51,18 +51,18 @@
  */
 #define _PATH_ATALKDEBUG	"/tmp/atalkd.debug"
 #define _PATH_ATALKDTMP		"atalkd.tmp"
-#define _PATH_ATALKDLOCK	ATALKPATHCAT(_PATH_LOCKDIR,"atalkd")
+#define _PATH_ATALKDLOCK	ATALKPATHCAT(_PATH_LOCKDIR,"atalkd.pid")
 
 /*
  * psorder paths
  */
 #define _PATH_TMPPAGEORDER	"/tmp/psorderXXXXXX"
-#define _PATH_PAPDLOCK		ATALKPATHCAT(_PATH_LOCKDIR,"papd")
+#define _PATH_PAPDLOCK		ATALKPATHCAT(_PATH_LOCKDIR,"papd.pid")
 
 /*
  * afpd paths
  */
 #define _PATH_AFPTKT		"/tmp/AFPtktXXXXXX"
-#define _PATH_AFPDLOCK		ATALKPATHCAT(_PATH_LOCKDIR,"afpd")
+#define _PATH_AFPDLOCK		ATALKPATHCAT(_PATH_LOCKDIR,"afpd.pid")
 
 #endif /* atalk/paths.h */
