--- netatalk.cvs/contrib/Makefile.am	2001/04/10 18:57:50	1.5
+++ netatalk.debian/contrib/Makefile.am	2001/05/05 16:17:40
@@ -1,11 +1,11 @@
 # Makefile.am for contrib/
 
-if COMPILE_TIMELORD
-TIMELORD = timelord
-else
-TIMELORD =
-endif
+#if COMPILE_TIMELORD
+#TIMELORD = timelord
+#else
+#TIMELORD =
+#endif
 
-SUBDIRS = nu printing shell_utils ${TIMELORD}
+SUBDIRS = nu printing shell_utils timelord #${TIMELORD}
 
 EXTRA_DIST = ICDumpSuffixMap
